lang/cs/BUILD.md

```diff
@@ -61,6 +63,42 @@ To run .NET tests on local machine from command line, execute
 
 `TestRunner.proj` already has a filter set up to exclude Yarn tests.
 
+***For Building DotNet projects:***
+
+We are in the process of converting the C# support to use dotnet core as well as support Visual Studio 2017. As a result 
+there are <projectname>.DotNet.csproj files that sit beside the existing csproj files. To build the dotnet projects, you will need 
+the .NET Core SDK installed on your machine. Note that Visual Studio 2017 is not required to build the files as the .NET Core SDK
+provides command line tools (referred to as dotnet-cli tools) that are required for building.
+
+Even though dotnet is supported on both windows and linux, the projects will only build fully on windows since they do target .net45. 
+The .NET Core SDK is located on github here:
+
+[https://github.com/dotnet/cli](https://github.com/dotnet/cli)
+
+Scroll down the page to access the latest builds of .NET Core SDK for your platform and download the .zip file. Create a `dotnet` 
+folder in your `Program Files` directory and extract the zip. Add `Program Files\dotnet` to your system path. If its working 
```

Comment:
Yes - the installer does do that, however I am recommending using the zip as I have found it easier to manage upgrades of dotnet. That way as newer versions of dotnet become available you dont have to uninstall -- you can just delete whatever is in the dotnet folder and extract the new bits. 

If you feel that installer is more streamlined (prevents mistakes, etc) and the way to go, Ill update the directions.